tools/build-tools/src/repoPolicyCheck/handlers/lockfiles.ts

```diff
@@ -13,7 +13,7 @@ const urlPattern = /(https?[^"@]+)(\/@.+|\/[^/]+\/-\/.+tgz)/g;
 
 export const handler: Handler = {
     name: "package-lockfiles",
-    match: /^.*?package-lock\.json$/i,
+    match: /^.*?[^_]package-lock\.json$/i, // Ignore _package-lock.json
```

Comment:
Covered by https://github.com/microsoft/FluidFramework/pull/5170, will need to merge that then update build tools version in this PR...